apps/dapp-console/app/components/Header/HeaderTabs.tsx

```diff
@@ -72,10 +81,17 @@ const SupportDropdownMenu = () => {
   const dropdownItemClasses =
     'flex items-center gap-2 cursor-pointer h-12 px-4 text-base text-secondary-foreground'
 
+  const handleDropdownOpenChange = (isOpen: boolean) => {
```

Comment:
would wrap this in a `useCallback` so it always has the latest reference to `setIsDropdownOpen`